tests/test_parse_to_object.py

```
@@ -416,6 +459,71 @@ def test_encode_parse_actuator_cmd_metadata(self):
+
+        bit_data = BitString()
+        bit_data.push(*TIMESTAMP_2.encode(0.0))  # Numeric.encode has float-precision issues at non-zero scaled values
+        bit_data.push(*Enum('cmd_state', 8, mt.actuator_state).encode('ACT_STATE_ON'))
+        msg_data = bit_data.pop(bit_data.length)
+
+        parsed = _ParsleyParseInternal.parse_to_object(msg_sid, msg_data)
+        assert isinstance(parsed, ParsleyObject)
+        assert parsed.msg_metadata == 0xFF  # fell back to int
+
+        # Round-trip the int-valued metadata
+        re_sid, re_data = _ParsleyParseInternal.encode_data(parsed.model_dump())
```

Ruff (0.15.13)

[warning] Unpacked variable `re_sid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)